src/rules/no-compositing-layer-props.ts

```
@@ -0,0 +1,133 @@
+        value.includes('scale3d') ||
+        value.includes('translateZ');
+    }
+
+    function checkProperty(propertyName: string, propertyValue?: string): boolean {
+      const normalizedName = normalizePropertyName(propertyName);
+      if (COMPOSITING_PROPERTIES.has(normalizedName)) {
+        // Special case for opacity - only warn if it's animated or fractional
+        if (normalizedName === 'opacity') {
+          if (!propertyValue) return false;
+          const numValue = Number.parseFloat(propertyValue);
+          if (isNaN(numValue)) return false;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)